packages/blade/src/components/Typography/Heading/__tests__/Heading.native.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.native';
+import Heading from '../';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<Heading />', () => {
+  it('should render Heading with variant "small"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { toJSON, getByText } = renderWithTheme(
+      <Heading type="normal" variant="small" weight="regular">
+        {displayText}
+      </Heading>,
+    );
+    expect(getByText('Get Started With Payment Gateway')).toBeTruthy();
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render Heading with variant "medium"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { toJSON, getByText } = renderWithTheme(
+      <Heading type="muted" variant="medium" weight="regular">
+        {displayText}
+      </Heading>,
+    );
+    expect(getByText('Get Started With Payment Gateway')).toBeTruthy();
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render Heading with variant "subheading" and weight "bold"', () => {
+    const displayText = 'Get Started With Payment Gateway';
+    const { toJSON, getByText } = renderWithTheme(
+      <Heading type="subdued" variant="large" weight="regular">
+        {displayText}
+      </Heading>,
+    );
+    expect(getByText('Get Started With Payment Gateway')).toBeTruthy();
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render Heading with variant "large"', () => {
```

Comment:
Move this case before the `should render Heading with variant "subheading" and weight "bold"` so that we have small, medium, large tests in sequence?

Reply:
copy paste 🤦